src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -266,7 +280,7 @@ private String waitFor() throws InterruptedException {
             Logger.debug(
                 this, "#waitFor(): process finished: %s", this.process
             );
-            if (!done.await(2L, TimeUnit.SECONDS)) {
```

Comment:
I think it means exactly the same: we are waiting for all monitors (N_MONITORS) when they are done.